authors/apps/articles/models.py

```diff
@@ -28,26 +27,10 @@ class Article(models.Model):
         related_name='articles',
         on_delete=models.CASCADE)
 
+    def __str__(self):
+        return self.title
+
     def save(self, *args, **kwargs):
         if not self.slug:
-            self.slug = self.get_unique_slag('title', 'slug')
+            self.slug = get_unique_slug(self, self.title, 'title', 'slug')
         super().save(*args, **kwargs)
-
-    def get_unique_slag(self, slugable_field_name, slug_field_name):
-        slug = slugify(self.title)
-        unique_slug = slug
-        # Use a hash string to create a unigue extension to
-        # append to the slug inorder to create a unique slag
-        extention = 1
-        ModelClass = self.__class__
-        # Search through the existing slags to check whether
-        # the slag we created above already exists. If it does,
-        # make it unique by appending a hyphen and the extension
-        # to it.
-        while ModelClass.objects.filter(
```

Comment:
Please help me understand your reason for removing this section that makes the slug unique in case its  duplicated?

Reply:
Thanks, @NicholusMuwonge for reviewing. The function was not removed but only its location was changed so that it can be reused